addons/l10n_es/data/account_fiscal_position_template_data.xml

```diff
@@ -521,6 +653,30 @@
             <field name="tax_src_id" ref="account_tax_template_s_iva21isp"/>
             <field name="tax_dest_id" ref="account_tax_template_s_req52"/>
         </record>
+        <record id="fptt_recargo_buy_0b"
+            model="account.fiscal.position.tax.template">
+            <field name="position_id" ref="fp_recargo"/>
+            <field name="tax_src_id" ref="account_tax_template_p_iva0_bc"/>
```

Comment:
@dylankiss I have detected that these lines should use `account_tax_template_p_iva0_s_bc` instead of `account_tax_template_p_iva0_bc` ("soportado 0%" instead of "exento").